Order/src/main/java/com/ws/Order/controller/AdminAssigController.java

```diff
@@ -0,0 +1,90 @@
+package com.ws.Order.controller;
+
+import com.ws.Order.models.AdministratorAssignments;
+import com.ws.Order.models.MasterAssignments;
+import com.ws.Order.models.Orders;
+import com.ws.Order.models.PartsList;
+import com.ws.Order.service.AdminAssigService;
+import com.ws.Order.service.MastersAssigService;
+import com.ws.Order.service.OrderService;
+import com.ws.Order.service.PartsListService;
+import org.aspectj.weaver.ast.Or;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+@RestController
+@RequestMapping("/api/Order")
+public class AdminAssigController {
+
+        @Autowired
+        private OrderService orderService;
+        private AdminAssigService adminAssigService;
+        private MastersAssigService mastersAssigService;
+        private PartsListService partsListService;
+
+        @GetMapping("")
+        public List<Orders> getAllOrder() {
+            return orderService.getAllOrder();
+        }
+
+        @GetMapping("/{id}")
+        public ResponseEntity<Orders> getOrderById(@PathVariable Long id) {
+            Orders account = orderService.getOrderById(id);
+            if (account != null) {
+                return new ResponseEntity<>(account, HttpStatus.OK);
+            } else {
+                return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+            }
```

Comment:
In such cases we do not need the else statement. If we were in the previous statement, we have definitely returned, so the else-case may be deleted, like
```suggestion
    if (account != null) {
        return new ResponseEntity<>(account, HttpStatus.OK);
    }
    return new ResponseEntity<>(HttpStatus.NOT_FOUND);
}
```